modules/govuk/manifests/deploy/config.pp

```diff
@@ -84,6 +92,11 @@
     require => File['/etc/govuk'],
   }
 
+  $csp_report_only_value = $csp_report_only ? {
+    true => 'yes',
```

Comment:
Minor, but would it be safer, given the ternary, to default this to 'yes', in case someone makes a config error? That said, perhaps it's better to expose config issues!

Reply:
This defaults to false at the parameter level so I think it's good.

Reply:
Indeed, I saw that. My concern was if someone tried to set the parameter to ‘yes’ it would be interpreted as ‘no’. I’m probably being over-cautious, though - the smaller the change, the smaller the edge case ;-).